atd-etl/app/process/helpers_import.py

```diff
@@ -565,22 +568,37 @@ def insert_crash_change_template(new_record_dict, differences, crash_id):
     return output
 
 
-def record_compare_hook(line, fieldnames, file_type):
+def record_crash_compare(line, fieldnames, crash_id, record_existing):
     """
     Hook that finds an existing record, and compares it with a new incoming record built from a raw csv line.
     :param line: string - The raw csv line
     :param fieldnames: array of strings - The strings to be used as headers
-    :param file_type: string - The file type (crash, units, charges, etc...)
     :return:
     """
-    if file_type == "crash":
-        fieldnames = [column.lower() for column in fieldnames]
-        crash_id = get_crash_id(line)
+    # Gather the field names
+    fieldnames = [column.lower() for column in fieldnames]
+
+    # If compare crash record is disabled, then exit.
+    if ATD_ETL_CONFIG["ATD_CRIS_IMPORT_COMPARE_FUNCTION"] != "ENABLED":
+        return False
+
+    # Double check if the record is valid, if not exit.
+    if record_existing is None:
+        return True
+
+    # The record is valid, it needs queueing or an upsert.
+    else:
+        # First generate a new crash record object from line
         record_new = generate_crash_record(line=line, fieldnames=fieldnames)
-        record_existing = get_crash_record(get_crash_id(line))
+        # Now calculate the differences
         differences = record_compare(record_new=record_new, record_existing=record_existing)
 
-        if len(differences) > 0:
+        # If there are no differences, just ignore...
+        if len(differences) == 0:
+            return False
+
+        # There are differences...
+        if record_existing["apd_human_update"] == "Y":
```

Comment:
I might be misunderstanding, but this reads to me like the only field we're checking as a proxy for knowing a record has been manually updated is `apd_human_update`. In the triggers, that only gets flipped to yes based on differences between `apd_confirmed_death_count` & `death_cnt`. 

Wouldn't we also want to check for `geocode_provider = 5` (aka Manual Q/A) so we flag human edits to coordinates too? Are there other cases?

Are there other clues for manual human edits?